frontend/lendme/lib/screens/other/add_item.dart

```diff
@@ -1,26 +1,206 @@
+import 'dart:io';
+import 'package:firebase_storage/firebase_storage.dart';
 import 'package:flutter/material.dart';
+import 'package:cloud_firestore/cloud_firestore.dart';
+import 'package:firebase_auth/firebase_auth.dart';
+import 'package:image_picker/image_picker.dart';
+import 'package:permission_handler/permission_handler.dart';
 
-class AddItem extends StatelessWidget {
+class AddItem extends StatefulWidget {
   const AddItem({Key? key}) : super(key: key);
 
+  @override
+  _AddItemState createState() => _AddItemState();
+}
+
+class _AddItemState extends State<AddItem> {
+  final FirebaseAuth firebaseAuth = FirebaseAuth.instance;
+  CollectionReference items = FirebaseFirestore.instance.collection('items');
```

Comment:
It would be better to encapsulate all firebase related stuff inside repositories. item_repository.dart already exists, so try to add "add_item" method there to avoid doing it here